docs/conf.py

```diff
@@ -61,10 +62,11 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
+import lasagne
 # The short X.Y version.
-version = '0.1'
+version = '.'.join(lasagne.__version__.split('.', 3)[:2]).rstrip('dev')
 # The full version, including alpha/beta/rc tags.
-release = '0.1'
+release = lasagne.__version__
```

Comment:
This saves another version string update that could easily be missed; numpy does it the same way. Let's see how readthedocs thinks about that. I've read that you may need to configure readthedocs to use `virtualenv`.